test/transaction_builder.js

```diff
@@ -206,7 +210,7 @@ describe('TransactionBuilder', function () {
     })
   })
 
-  var networksToTest = ['bitcoin', 'bitcoingold', 'bitcoincash', 'zcashTest']
+  var networksToTest = ['bitcoin', 'bitcoingold', 'bitcoincash', 'bitcoinsv', 'zcashTest']
```

Comment:
Why is zcash the only testnet in this list?

Reply:
Because when I added Zcash to the lib I was using data from testnet. In this particular test, it does not make a difference, but I'll change it to the main network to be consistent. Also, I'm adding litecoin and dash